template/web/wp-content/themes/<%= slug %>/app/Repositories/Repository.php

```diff
@@ -91,7 +91,14 @@ protected function query( array $params, $class_type ) {
 		return $this->result_set( $results );
 	}
 
-	abstract function do_query( $params, $class_type );
+	/**
+	 * Function to implement in an extended Repository
```

Comment:
Maybe explain the intent and usage of this method here so it is easier for developers to implement it.

Reply:
I am not sure what i should add, abstract functions must be defined when you extend a class

Reply:
What is the purpose of this method? Why does it need to be implemented? What should it return? All the questions a developer not knowing the class might need to know in order to implement this method correctly.